src/index.rst

```diff
@@ -107,12 +107,17 @@ Component Level Dependencies
 
 Axom has the following inter-component dependencies:
 
-- Core has no dependencies and the other components depend on Core
-- Slic optionally depends on Lumberjack
-- Slam, Spin, Primal, Mint, Quest, and Sidre depend on Slic
-- Mint optionally depends on Sidre
-- Quest depends on Slam, Spin, Primal, and Mint
-- Inlet depends on Sidre, Slic, and Primal
+- Core, Slic, and Lumberjack provide basic services to the rest of Axom and to user code
```

Comment:
> Core, Slic, and Lumberjack provide basic services to the rest of Axom and to user code

Should we revert this one as well?
I'm not sure I agree with this characterization.

Reply:
I'm interpreting "Should we revert this one as well?" as "Please revert this one as well."  I'm a little startled, since this change and the change to the dependency graph appeared to be a mundane refactoring (a decrease in complexity and no change in functionality).  Since apparently the changes don't make a refactoring, the best thing will be to discuss them further.